### DISCUSSION TITLE
How can I change the map in TeslaMate? Google Maps cannot be displayed in China, how can I replace the code to use a domestic map API? Thank you.

### DISCUSSION THREAD

**Discussion by bloy123** [teslamate-org/teslamate#3860] [General]
How can I change the map in TeslaMate? Google Maps cannot be displayed in China, how can I replace the code to use a domestic map API? Thank you.

**Comment by blue-eric** [comment]
关于中国使用地图问题，好像并没有更多的人原因分享修改方法

**Comment by Amourgo** [comment]
Is there a solution to switch to a Chinese map service provider?
